blocks/elastic-content-strip/elastic-content-strip.js

```diff
@@ -0,0 +1,26 @@
+import { a } from '../../scripts/dom-helpers.js';
+
+import {
+  decorateInnerHrefButtonsWithArrowIcon,
+  decorateTemplateAndTheme,
+} from '../../scripts/scripts.js';
+
+export default function decorate(block) {
+  block.querySelectorAll('.elastic-content-strip > div > div').forEach((div) => {
+    const elasticContentWrapper = a({
+      class: 'elastic-content-link-wrapper',
+      href: div.children[3].children[0].href,
```

Comment:
Rather than access it like this, can you use a querySelector instead? This way you ensure you get the link no matter the position.